.github/actions/ovsx-publish/main.js

```diff
@@ -29,10 +31,13 @@ const {
     extensionFile: ARTIFACT_FILE,
     pat: OPENVSX_TOKEN
   };
-  await ovsx.publish(options);
-
+  const [ publicationResult ] = await ovsx.publish(options);
```

Comment:
This is what caused the silent failure: `ovsx.publish` returns an array of `PromiseSettledResult` instead of failing 🤦🏻

Reply:
If I'm looking at the [correct thing](https://github.com/eclipse/openvsx/blob/f4b5687f062ada70bfed41cf55ebf36a4b50bb6b/cli/src/publish.ts#L18), `publish()` returns a single `Promise<PromiseSettledResult<void>[]>`. With the `await`, we will get the array of settled/unsettled promises.

So I am thinking that we should consider the full array (not only the first element), and check if any of them have been rejected in the `if` below. 🤔

Reply:
In theory, yes; and when I run the script, the array contains only a single result. It could contain several items if we were publishing several packages at once.

Reply:
OK, that makes sense!